src/main/java/me/figo/FigoSession.java

```diff
@@ -458,17 +316,16 @@ public void setAccountOrder(List<Account> orderedList) throws FigoException, IOE
      * @param count
      *            limit the number of returned transactions
      * @param offset
-     *            which offset into the result set should be used to determin the first transaction to return (useful in combination with count)
+     *            which offset into the result set should be used to determine the first transaction to return (useful in combination with count)
      * @param include_pending
      *            this flag indicates whether pending transactions should be included in the response; pending transactions are always included as a complete
      *            set, regardless of the `since` parameter
      * @return an array of Transaction objects
      */
-    public List<Transaction> getTransactions(Account account, String since, Integer count, Integer offset, Boolean include_pending) throws FigoException,
-            IOException {
+    public List<Transaction> getTransactions(Account account, String since, Integer count, Integer offset, PendingTransactions include_pending) throws FigoException, IOException {
```

Comment:
Because when I see a line like this
getTransactions(..., true)
I have to look at the code or documentation for what the last parameter is good for.
But it is code style not an error.

http://programmers.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior